tools/DfmHttpService/Handler/HandlerServerException.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmHttpService
+{
+    public class HandlerServerException : HandlerException
+    {
+        public HandlerServerException() : this("Error happens while handling http context")
```

Comment:
>Error happens while handling http context [](start = 48, length = 41)

the error message should be different from base and HandlerClientException? or could call base ctor